rakelib/plugin.rake

```diff
@@ -61,6 +61,16 @@ namespace "plugin" do
     task.reenable # Allow this task to be run again
   end
 
+  task "install-default-oss" => "bootstrap" do
```

Comment:
review note: I am okay to add `[:oss]` as an argument and exclude OSS except plugins but I prefered to separate to make clearly visible. When we add an argument to the task, multiple (ex: `prepare` -> `install-default`) tasks should carry this argument.